### PR TITLE
Add Meagcoin (MEC)

### PR DESCRIPTION
**Pull request by dalijolijo**
Adding Megacoin (MEC)

Website: https://www.megacoin.eu
SLIP44= 217
COIN_P2PKH_VERSION=50
COIN_P2SH_VERSION=5

### CODE REVIEW

File: include/keepkey/firmware/coins.def [hunk 1]
```diff
@@ -31,5 +31,6 @@ X(true, "Electra",      true, "ECA",   true,  33,    true,      10000,  true,  4
...
 X(true, "GRS Testnet",  true, "tGRS",  true, 111,    true,     100000 , true, 196,  true, "\x1c" "GroestlCoin Signed Message:\n",   true, 0x80000001,   false,   0,     true,   8,  false, NO_CONTRACT,      true, 70617039,  true, true,   true, false,  true,"secp256k1-groestl",          false, "",            true, "tgrs",  false, false,  true,  71979618,        true, 73342198,            false, ""    )
+X(true, "Megacoin",     true, "MEC",   true,  50,    true,    2000000,  true, 196,  true, "\u0019MegaCoin Signed Message:\n",       true, 0x800000d9,   false,   0,     true,   8,  false, NO_CONTRACT,      true, 76067358,  true, false,  true, false,  true, SECP256K1_STRING,            false, "",            false, "",     false, false,  false,        0,        false,       0,            false, ""    )
```
Comment (keepkeyjon) [inline include/keepkey/firmware/coins.def]:
```suggestion
X(true, "Megacoin",     true, "MEC",   true,  50,    true,    1000000,  true,   5,  true, "\x19" "MegaCoin Signed Message:\n",      true, 0x800000d9,   false,   0,     true,   8,  false, NO_CONTRACT,      true, 76067358,  true, false,  true, false,  true, SECP256K1_STRING,            false, "",            false, "",     false, false,  true,  77429938,        true, 78792518,            false, ""    )
```

Comment (dalijolijo) [reply include/keepkey/firmware/coins.def]: thank you.

Comment (keepkeyjon) [reply include/keepkey/firmware/coins.def]: I think you need to hit the commit button on the suggestion... I can't.

**Comment by dalijolijo** [comment]
commit done

**Pull request merged**